compiler/modules/front/codeGen.bal

```diff
@@ -1310,15 +1309,11 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
         var { td, left, negated, kwPos:pos } => {
             return codeGenTypeTest(cx, bb, env, td, left, negated, pos);
         }
-        // Variable reference
-        // JBUG #33309 does not work as match pattern
-        var ref if ref is s:VarRefExpr => {
-            return codeGenVarRefExpr(cx, ref, env, bb);
+        var {prefix, name, startPos} => {
+            return codeGenVarRefExpr(cx, env, bb, prefix, name, startPos);
         }
-        // Constant
-        // JBUG #33309 does not work as match pattern `var { value, multiSemType }`
-        var cvExpr if cvExpr is s:ConstValueExpr => {
-            return codeGenConstValue(cx, bb, env, cvExpr);
+        var { value, multiSemType, startPos:pos } => {
```

Comment:
space after colon

Reply:
I will send a separate PR for this one as there are lots of similar places to fix.